tpe/lib/src/Link.hh

```diff
@@ -42,6 +44,39 @@ class IGNITION_PHYSICS_TPELIB_VISIBLE Link : public Entity
   /// \brief Add a collision
   /// \return Newly created Collision
   public: Entity &AddCollision();
+
+  /// \brief Set the linear velocity of link
+  /// \param[in] _velocity linear velocity
+  public: void SetLinearVelocity(const math::Vector3d &_velocity);
+
+  /// \brief Get the linear velocity of link
+  /// \return linear velocity of link
+  public: math::Vector3d GetLinearVelocity() const;
+
+  /// \brief Set the angular velocity of link
+  /// \param[in] _velocity angular velocity
+  public: void SetAngularVelocity(const math::Vector3d &_velocity);
+
+  /// \brief Get the angular velocity of link
+  /// \return angular velocity
+  public: math::Vector3d GetAngularVelocity() const;
+
+  /// \brief Update the pose of the entity
+  /// \param[in] _timeStep current world timestep
```

Comment:
It would be nice to document the unit. Alternatively, we could use chrono so there's no ambiguity.

Reply:
added unit. 58f3e0b